code/modules/clothing/under/syndicate.dm

```diff
@@ -70,12 +70,32 @@
 	inhand_icon_state = "bl_suit"
 	has_sensor = HAS_SENSORS
 	armor_type = /datum/armor/syndicate_tacticool
+	stubborn_stains = TRUE
 
 /datum/armor/syndicate_tacticool
 	bio = 10
 	fire = 50
 	acid = 40
 
+/obj/item/clothing/under/syndicate/tacticool/examine(mob/user)
+	. = ..()
+	. += "It has a label that says cleaning this 'genuine' Waffle Corp. product with cleaning solutions other than Grime Liberator telelocational podcrystals will void the warranty."
+	. += "What on earth is a <font color='red'>tele</font>locational pod<font color='red'>crystal</font>?"
+
+/obj/item/clothing/under/syndicate/tacticool/dye_item(dye_color, dye_key_override)
+	if(dye_color == DYE_SYNDICATE)
+		if(dying_key == DYE_REGISTRY_JUMPSKIRT)
+			special_wash(/obj/item/clothing/under/syndicate/skirt)
+		else
+			special_wash(/obj/item/clothing/under/syndicate)
+		qdel(src)
+		return
+	else
+		return ..()
```

Comment:
```suggestion
	if(dye_color != DYE_SYNDICATE)
		return ..()

	if(dying_key == DYE_REGISTRY_JUMPSKIRT)
		special_wash(/obj/item/clothing/under/syndicate/skirt)
	else
		special_wash(/obj/item/clothing/under/syndicate)
	qdel(src)
```

way more simplistic and clear

Reply:
I think that call to parent is so that you can dye the jumpsuit with other types of colors, rather than the special dye from telecrystals. This code would make it impossible to dye the jumpsuit with anything _but_ the telecrystal.

Reply:
alright then just do this then

```suggestion
	if(dye_color == DYE_SYNDICATE)
		if(dying_key == DYE_REGISTRY_JUMPSKIRT)
			special_wash(/obj/item/clothing/under/syndicate/skirt)
		else
			special_wash(/obj/item/clothing/under/syndicate)
		qdel(src)
		return

	return ..()
```